cvat/settings/base.py

```diff
@@ -666,5 +668,6 @@ class CVAT_QUEUES(Enum):
                 'key': ''
             },
             'SCOPE': [ 'read:user', 'user:email' ],
+            'PUBLIC_NAME': 'GitHub',
```

Comment:
I will recommend adding a comment here. It is something that you have added to especially for our needs. It is the right place to specify why we need that and how it will behave if the user doesn't provide the field.